howdoi/howdoi.py

```diff
@@ -397,17 +430,20 @@ def howdoi(raw_query):
 
 
 def get_parser():
-    parser = argparse.ArgumentParser(description='instant coding answers via the command line')
+    parser = argparse.ArgumentParser(
+        description='instant coding answers via the command line')
```

Comment:
There's no need to break this line. We are using 119 length as the max https://github.com/gleitz/howdoi/blob/master/.pylintrc#L321